src/MongoDB.Analyzer/Core/Builders/BuildersAnalyzer.cs

```diff
@@ -68,6 +69,9 @@ private static AnalysisStats ReportMqlOrInvalidExpressions(MongoAnalysisContext
         var settings = context.Settings;
         int mqlCount = 0, internalExceptionsCount = 0, driverExceptionsCount = 0;
 
+        var typesMapper = new TypesMapper();
+        typesMapper.AddMappings(MqlGeneratorSyntaxElements.Builders.MqlGeneratorNamespace, context.TypesProcessor);
```

Comment:
Lets move `AddMappings` to `TypesMapper` ctor for simplicity. And create it only when needed:
`typesMapper ?? = new TypesMapper();`.

Reply:
Done